code/modules/clothing/suits/wiz_robe.dm

```diff
@@ -149,8 +154,13 @@
 
 /obj/item/clothing/head/wizard/marisa/fake
 	name = "witch hat"
-	desc = "Strange-looking hat-wear, makes you want to cast fireballs."
-	icon_state = "marisa"
+	desc = "Strange-looking hat-wear. Makes you want to cast fireballs."
+	icon_state = "witch_hat"
+	greyscale_colors = "#343640#e0cab8#e0cab8"
+	greyscale_config = /datum/greyscale_config/witch_hat
+	greyscale_config_worn = /datum/greyscale_config/witch_hat_worn
+	flags_1 = IS_PLAYER_COLORABLE_1
+	dog_fashion = null
```

Comment:
None of this has to be here because it's a subtype of ``/obj/item/clothing/head/wizard/marisa`` so copies it from that

Reply:
Removed the surplus code and tested to ensure it works still. Thank you!